Sources/NIOSSL/SSLCertificate.swift

```diff
@@ -39,7 +48,7 @@ public class NIOSSLCertificate {
     }
 
     private init(withReference ref: UnsafeMutablePointer<X509>) {
-        self.ref = ref
+        self._ref = .init(ref)
```

Comment:
While I appreciate the desire to make this concise, when it comes to working with the pointer types I really prefer being able to see what the actual type is at the point of initialisation, rather than finding the declaration. Can we use the longer form initializer here?